rpc/backend/utils.go

```diff
@@ -291,3 +294,22 @@ func GetHexProofs(proof *crypto.ProofOps) []string {
 	}
 	return proofs
 }
+
+// HandleBroadcastRawLog checks the RawLog for known temporary rejection or nonce gap error.
+// NOTE: make sure it sync with the latest go-ethereum logic when upgrade:
+// https://github.com/ethereum/go-ethereum/blob/master/core/txpool/locals/errors.go#L29
+func HandleBroadcastRawLog(rawLog string, txHash common.Hash) (bool, string) {
+	if strings.Contains(rawLog, mempool.ErrNonceGap.Error()) {
+		return true, "transaction queued due to nonce gap"
+	}
+	switch rawLog {
+	case legacypool.ErrOutOfOrderTxFromDelegated.Error(),
```

Comment:
Wouldn't these get outright rejected? I'm not sure I understand what temporary rejection means in this setting. The reason we treat nonce gaps differently is because they get added into the queue on the EVM side despite being rejected on the Cosmos side, so we don't want to return errors on those.

However, if a transaction is underpriced, there are nonce gaps on 7702 accounts, etc., these get discarded from the pool whatsoever, behaving differently from our nonce gap handling.

Could you reference the geth code that returns a hash on these and somewhere where we I read up on temporary rejections?

Reply:
Looks like I just found it: https://github.com/cosmos/go-ethereum/blob/9089f9461cc2a25703ee247256e1d0c48b64f815/core/txpool/locals/errors.go#L29-L28

Could we use this function instead? Should improve maintainability